README.md

```diff
@@ -1,65 +1,71 @@
+* Verify that you can enter ssh into this machine without a password using `ssh root@1.2.3.4`
+* Ensure that you have a recent version of nix (we use 2.7, but from 2.5 upwards will probably be okay)
+* Ensure that your `nixops --version` responds with something like `NixOps 2.0.0-pre-7220cbd`
+* Adapt in machines/hello.nix the value of `1.2.3.4` in the targetHost definition
+* Change in machines/hello.nix "your ssh key" to your public key
 
-It is also possible to install the unstable version using nixopsUnstable where `nixops --version` returns `NixOps 2.0.0`.
+Now test everything with the following commands on you shell
 
-nixops allows to use different backends and mix them. This guide also describes how to use the libvirt backend to quickly deploy to qemu using libvirtd, and the none backend to deploy pre-made VPSs over SSH.
+* `git clone https://github.com/vpsfreecz/example-nixops-deployment.git`
+* `cd example-nixops-deployment`
+* `nixops create -d hello`
+* `nixops deploy -d hello --test`
+* Verify that you can enter via `ssh root@1.2.3.4` and have the commands like vim, git, fish installed
+* `nixops deploy -d hello --boot --force-reboot` to activate your machine
 
-# Example
+If we do not specify the `NIXOPS_DEPLOYMENT` environment variable, we need to use the `-d` parameter and specify the name of the deployment.
 
-We start by cloning the repository with a sample deployment:
+## Modifications done compared to a minimum
 
-`git clone https://github.com/vpsfreecz/example-nixops-deployment/`
+### in `machines/hello.nix`
 
-The deployment in `network.nix` defines two machines named hello and world that use include to import configuration from the machines directory. The `machines/hello.nix` configuration shows the deployment of the nginx webserver and the `machines/world.nix` configuration shows the PostgreSQL enablement.
+* Addded `imports = [ ../vpsadminos.nix ];`
+* Disabled creating the manual, which takes a long time by setting `documentation.nixos.enable = false;`
+* Enable OpenSSH to be able login after reboot via `services.openssh.enable = true;`
 
-## Testing using libvirt backend
+### in flake.nixops
 
-The sample deployment can be tested using the libvirt backend. Use `nixops create` to create a new deployment called virt, consisting of network.nix (logical configuration) and network-libvirt.nix (physical configuration, specific to libvirt).
+* Added `network.enableRollback = true;` for rollback. Use it `nixops list-generations -d hello`
 
-We can deploy this deployment using the deploy command. In the case of the libvirt backend, deploy will produce the defined virtual machines and upload the new configuration to them.
+### in vpsadminos.nix
 
-If we do not specify the `NIXOPS_DEPLOYMENT` environment variable, we need to use the `-d` parameter and specify the name of the deployment.
+Here we might have problems as I think the definitions for documentation and
+openssh might contradict with the settings in machine/hello.nix.
```

Comment:
This should not be a problem, because vpsadminos.nix defines the settings as defaults. You can override them without issues, so this can be deleted.